### DISCUSSION TITLE
Welcome to Eradiate Discussions!

### DISCUSSION THREAD

**Discussion by leroyvn** [eradiate/eradiate#13] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Ygovs** [comment]
tinydb package is required for the 1D tutorial

**Reply by leroyvn** [reply]
Hi @Rayference, this was actually an issue with dependency handling. It should be fixed now.

**Comment by leroyvn** [comment]
I'm Vincent Leroy. I work for Brussels-based company Rayference and I'm a former academic scientist specialised in porous material modelling. I am the lead developer of Eradiate since 2017.
